test/e2e/e2e_leaderelection_test.go

```diff
@@ -124,10 +157,18 @@ func TestLeaderElection(t *testing.T) {
 	}
 	t.Logf("Removed kube-system/descheduler lease")
 
+	tc := struct {
```

Comment:
use single tc here instead of the tests []struct in #1472 
Not sure if it's better to use []struct, includes the cases of ns1 and ns2 then run
But I suppose ns1 and ns2 should belongs to the same test case

Reply:
There's currently only a single test to be run. With two descheduler pods run in the same namespace.